static/app/types/event.tsx

```diff
@@ -357,14 +357,29 @@ export type Entry =
   | EntryGeneric
   | EntryResources;
 
-// Contexts
-type RuntimeContext = {
+// Contexts: https://develop.sentry.dev/sdk/event-payloads/contexts/
+
+export interface BaseContext {
+  type: string;
+}
+
+export enum RuntimeContextKey {
+  BUILD = 'build',
+  NAME = 'name',
+  RAW_DESCRIPTION = 'raw_description',
+  VERSION = 'version',
+}
+
+// https://develop.sentry.dev/sdk/event-payloads/contexts/#runtime-context
+export interface RuntimeContext
```

Comment:
are these exports being used?

Reply:
nope, I just exported them by default since I thought it would make it easier for users (don't have to add export statements in their PRs), but I can change this.

Reply:
I don't think we have a pattern for this...maybe this is a topic for Frontend TSC but I personally only add the export if it is used somewhere else